src/main/scala/com/microsoft/sqlserver/jdbc/spark/utils/BulkCopyUtils.scala

```diff
@@ -239,45 +261,71 @@ object BulkCopyUtils extends Logging {
         val dfCols = df.schema
 
         val tableCols = getSchema(rs, JdbcDialects.get(url))
+        val ccol = getComputedCols(conn, options.dbtable)
```

Comment:
Should get abstracted to a single function that just gives back a list of computed columns.

Reply:
Done